### PR TITLE
Fix responsive images rendering of conversions

### PR DESCRIPTION
**Pull request by simonellensohn**
At the moment the call to `hasResponsiveImages` in `img` only checks if there are any conversions on the current collection, ignoring the given conversion. In which case the rendered output of the conversion would not contain the `srcset` attribute.

This fix checks if there are any responsive images of the given conversion.

### CODE REVIEW

File: tests/Feature/Media/ToHtmlTest.php [hunk 1]
```diff
@@ -93,6 +93,19 @@ public function it_can_render_itself_with_responsive_images_and_a_placeholder()
...
+    /** @test */
+    public function it_can_render_itself_with_responsive_images_of_a_conversion_and_a_placeholder()
+    {
+        $media = $this->testModelWithResponsiveImages
+            ->addMedia($this->getTestJpg())
+            ->toMediaCollection();
+
+        $image = $media->refresh()->img('thumb');
+
+        $this->assertEquals(1, substr_count($image, '/media/2/responsive-images/'));
```
Comment (freekmurze) [inline tests/Feature/Media/ToHtmlTest.php]: Can we use `assertContains` here?

**Comment by simonellensohn** [comment]
I've replaced `assertEquals` with `assertContains`, feels a lot cleaner now.

File: tests/Feature/Media/ToHtmlTest.php [hunk 2]
```diff
@@ -93,6 +93,19 @@ public function it_can_render_itself_with_responsive_images_and_a_placeholder()
...
+    public function it_can_render_itself_with_responsive_images_of_a_conversion_and_a_placeholder()
+    {
+        $media = $this->testModelWithResponsiveImages
+            ->addMedia($this->getTestJpg())
+            ->toMediaCollection();
+
+        $image = $media->refresh()->img('thumb');
+
+        $this->assertContains('/media/2/responsive-images/', $image);
+        $this->assertTrue(str_contains($image, 'data:image/svg+xml;base64,'));
```
Comment (freekmurze) [inline tests/Feature/Media/ToHtmlTest.php]: Could you use `assertContains` here?

**Comment by freekmurze** [comment]
Thanks!

**Pull request merged**